pkg/client/clientset_generated/release_1_1/clientset.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package release_1_1
+
+import (
+	extensions_unversioned "k8s.io/kubernetes/pkg/client/typed/generated/extensions/unversioned"
+	legacy_unversioned "k8s.io/kubernetes/pkg/client/typed/generated/legacy/unversioned"
+	unversioned "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+type Interface interface {
+	Legacy() legacy_unversioned.LegacyClient
+	Extensions() extensions_unversioned.ExtensionsClient
+}
+
+// Clientset contains the clients for groups. Each group at most may have one
```

Comment:
What I mean is "you may have multiple groups in your pkg/apis/ directory, but you can choose to generate a clientset that doesn't include a particular group, so there would be zero version for that group". For example, this clientset doesn't include the `abac` group.

I guess my comment is a little misleading, because the user of clientset doesn't care about what's in pkg/apis/. What do you think?